### PR TITLE
Ingestion Controls: Agent Level configuration modal

### PR DESCRIPTION
**Pull request by dussault-antoine**
### What does this PR do?

Add documentation for the new Agent Level configuration modal
[JIRA Epic link](https://datadoghq.atlassian.net/browse/ATAFE-1769)

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by kayayarai** [comment]
Opened DOCS-3921 to track edit pass

File: content/en/tracing/trace_pipeline/ingestion_controls.md [hunk 1]
```diff
@@ -96,25 +114,25 @@ Click on any service row to view the Service Ingestion Summary, a detailed view
...
-To specify that a specific percentage of a service's traffic should be sent, add a generated code snippet to your tracer configuration for that service.
+To specify that a specific percentage of a service's traffic should be sent, add an environment variable or a generated code snippet to your tracer configuration for that service.
 
 1. Select the service you want to change the ingested span percent for.
 2. Choose the service language.
 3. Choose the desired ingestion percentage.
-4. Apply the appropriate configuration generated from these choices to the indicated service and redeploy.
-5. Confirm on the Ingestion Control Page that your new percentage has been applied. A share of the Ingestion Reason Breakdown should be allocated to `ingestion_reason:rule`.
+4. Apply the appropriate configuration generated from these choices to the indicated service and redeploy the service.
+5. Confirm on the Ingestion Control Page that your new percentage has been applied: the Traffic Breakdown column should surface the sampling rate applied. The ingestion reason for the service should also be `ingestion_reason:rule`.
```
Comment (kayayarai) [inline content/en/tracing/trace_pipeline/ingestion_controls.md:135]:
```suggestion
5. Confirm on the Ingestion Control Page that your new percentage has been applied by looking at the Traffic Breakdown column, which surfaces the sampling rate applied. The ingestion reason for the service is shown as `ingestion_reason:rule`.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Some edits, and a couple questions for you.

File: content/en/tracing/trace_pipeline/ingestion_controls.md [hunk 2]
```diff
@@ -23,54 +23,72 @@ further_reading:
...
+{{< img src="tracing/trace_indexing_and_ingestion/ingestion_controls_page.png" style="width:100%;" alt="Ingestion Control Page Overview" >}}
 
 All metrics used in the page are based on live traffic data of the **past 1 hour**. Any Agent or library configuration change is reflected in the page.
 
 ## Summary across all environments
 
 Get an overview of the total ingested data over the past hour, and an estimation of your monthly usage against your monthly allocation, calculated with the active APM intrastructure (hosts, Fargate tasks, and serverless functions).
 
-If the monthly usage is under `100%`, the projected ingested data fits in your [monthly per-host allotment][3]. A quota over `100%` means that the monthly ingestion volume is projected to be over your monthly allocation.
+If the monthly usage is under `100%`, the projected ingested data fits in your [monthly allotment][3]. A quota over `100%` means that the monthly ingestion volume is projected to be over your monthly allocation.
```
[the remark this answers is not in the document]
Comment (dussault-antoine) [reply content/en/tracing/trace_pipeline/ingestion_controls.md:41]: let's use monthly allotment everywhere
and ingested data

Comment (kayayarai) [inline content/en/tracing/trace_pipeline/ingestion_controls.md:41]:
```suggestion
If the monthly usage is under `100%`, the projected ingested data fits in your [monthly allotment][3]. A monthly usage value over `100%` means that the monthly ingested data is projected to be over your monthly allotment.
```

**Review by kayayarai — APPROVED** [review]

**Pull request merged**